datasets/lt/pep_declarations/crawler.py

```diff
@@ -0,0 +1,199 @@
+            if status_code := response.pop("status") == 404:
+                self.context.log.info(f"deklaracija {id_str} does not exist")
+            else:
+                self.context.log.error(
+                    f"deklaracija {id_str} skipped due to {status_code} error."
+                )
+
+
+def make_person(context: Context, data: dict) -> Entity:
+    person = context.make("Person")
+    first_name = data.pop("vardas")
+    last_name = data.pop("pavarde")
+    person_id = data.pop("asmensKodas", None)  # this identifier is often missing
+    person.id = context.make_id(person_id, first_name, last_name)
+    person.add("firstName", first_name)
+    person.add("registrationNumber", person_id)
+    person.add("lastName", last_name)
+    person.add("birthDate", data.pop("gimimoData", None))
+    person.add("legalForm", data.pop("asmensTipas", None))
+    context.audit_data(data)
+    return person
+
+
+def make_marriage(context: Context, person: Entity, spouse: Entity) -> Entity:
+    marriage = context.make("Family")
+    marriage.id = context.make_id(person.id, spouse.id)
+    marriage.add("relationship", "Spouse")
+    marriage.add("person", person)
+    marriage.add("relative", spouse)
+    return marriage
+
+
+def parse_affiliations(
+    context: Context, person: Entity, affiliations: list[dict], is_pep: bool = True
+) -> list[tuple[Entity]]:
+    """
+    Args:
+        context (Context)
+        affiliations (list[dict]): A list of dicts, where each dict refers to a place
+         of work and a sub-list of positions.
+        is_pep (bool, optional): Defaults to True.
+
+    Returns:
+        list[tuple[Entity]]: a flattened list of tuples, where each is
+        a position and occupancy. Occupancies that do not meet OpenSanctions criteria
+        are skipped.
+    """
+    entities = []
+    for data in affiliations:
+        entity_is_legal_entity: bool = data.pop("yraJuridinisAsmuo")
+        entity_is_lithuanian: bool = data.pop("registruotaLietuvoje")
+        legal_entity_code: Optional[str] = data.pop("jaKodas")
+        entity_name = data.pop("pavadinimas")
+        affiliation_start_date: str = data.pop("rysioPradzia")
+        affiliation_type = data.pop("darbovietesTipas")
+        data_sources = data.pop("duomenuSaltiniai")
+        filed_automatically: bool = data.pop("uzpildytaAutomatiskai")
+        entity_must_be_declared: bool = data.pop("privaluDeklaruoti")
+        entity_legal_form: str = data.pop("jaTeisinesFormosPavadinimas")
+
+        for role in data.pop("pareigos"):
+            position_name: Optional[str] = role.pop("pareigos")
+            legal_code: Optional[str] = role.pop("teisejoKodas")
+            role_must_be_declared: Optional[bool] = role.pop("privaluDeklaruoti")
+            nature_of_duties: str = role.pop("pareiguTipasPavadinimas")
+            position = h.make_position(
+                context,
+                name=position_name or "Unknown position",
```

Comment:
I think it's worth combining pareigos with pavadinimas so that we get roles like `Director, European Social Fund Agency`

https://zavod.opensanctions.org/peps/#selecting-a-position-name